snappy_pipeline/workflows/somatic_variant_annotation/__init__.py

```diff
@@ -149,10 +160,19 @@ def get_output_files(self, action):
         """Return output files for the filtration"""
         # Validate action
         self._validate_action(action)
-        prefix = (
-            "work/{{mapper}}.{{var_caller}}.{annotator}.{{tumor_library}}/out/"
-            "{{mapper}}.{{var_caller}}.{annotator}.{{tumor_library}}"
-        ).format(annotator=self.annotator)
+        tpl = f"{{mapper}}.{{var_caller}}.{self.annotator}"
+        if self.config.filtration_schema == FiltrationSchema.list:
+            tpl += ".filtered.{tumor_library}"
+        elif self.config.filtration_schema == FiltrationSchema.sets:
+            tpl += ".".join(
+                "dkfz_bias_filter.eb_filter",
+                "{tumor_library}",
+                self.config.filter_sets,
+                self.config.exon_lists,
+            )
+        elif self.config.filtration_schema == FiltrationSchema.unfiltered:
+            tpl += ".{tumor_library}"
+        prefix = os.path.join("work", tpl, "out", tpl)
```

Comment:
I think you may be able to extract this logic to a function and call that everywhere you make use of it

Reply:
Done